composer.json

```diff
@@ -27,8 +27,7 @@
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/orm": "^2.5",
         "doctrine/persistence": "^1.3.4",
-        "sonata-project/admin-bundle": "^3.61",
-        "sonata-project/core-bundle": "^3.14",
+        "sonata-project/admin-bundle": "^3.63 || ^4.0",
```

Comment:
Right now the tests pass, but what is actually installed is the admin bundle v3. Besides, there is already a job dedicated to testing with admin v4:

https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/8a7805f49ec9895517af44b8d6df5874d676f88e/.travis.yml#L54

Reply:
I know about it. This PR is for allow using(install) DoctrineORMAdminBundle(master) with AdminBundle 4.x(master). Before major release dev-kit should be update, travis here should be update. Composer should be update.  

New DoctrineORMAdminBundle should not support CoreBundle:
> https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/8a7805f49ec9895517af44b8d6df5874d676f88e/.travis.yml#L40

Reply:
> This PR is for allow using(install) DoctrineORMAdminBundle(master) with AdminBundle 4.x(master).

Why do you want to do that?

Reply:
Indeed, there no 4.0 version for the admin bundle so I do not get it...

Reply:
I'm working on AdminLTE 3 integration and webpack. I have to do it in master versions.

Reply:
AdminLTE 3 should be add to AdminBundle And all storage-AdminBundle

Reply:
You should be able to use the same sort of inline alias trick, `dev-master as 3.0.0` for admin bundle should work, but maybe that's somehow inconvenient?